tests/unit/vdom.ts

```diff
@@ -2708,4 +2709,38 @@ describe('vdom', () => {
 			});
 		});
 	});
+
+	describe('behavior', () => {
+		it('will append nodes with attributes already attached', (test) => {
+			if (!global.window || !global.window.hasOwnProperty('MutationObserver')) {
+				test.skip('MutationObserver not present');
+			}
```

Comment:
It’s not great to skip a test due to the test using something that is potentially incompatible yet the actual code is. I’m not convinced a mutation observer is actually needed anyway, can you not just stub createElement and the associated things off it?